src/org/usfirst/frc4904/robot/commands/OverridableCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+/**
+ * 
+ * @author niksure
+ *         This interface allows commands to easily override other commands
+ */
+public interface OverridableCommand {
+	/**
+	 * sets an override to an static boolean you give it
+	 * 
+	 * @param override
+	 */
+	void setOverride(boolean setValue);
+
+	/**
+	 * returns if override is on or off
+	 * 
+	 * @return
+	 */
+	boolean getOverride();
```

Comment:
Can we change this to `isOverridden` so we can make the other one `isNotOverridden`?